kahuna/public/js/search/results.js

```diff
@@ -248,6 +248,7 @@ results.controller('SearchResultsCtrl', [
             // FIXME: should ideally be able to just call $state.reload(),
             // but there seems to be a bug (alluded to in the docs) when
             // notify is false, so forcing to true explicitly instead:
+            window.scrollTo(0,0);
```

Comment:
I think this should be [`$window`](https://github.com/guardian/grid/pull/2876/files#diff-b8c6d4770b5b04294cc8c88d27ae65d3L78) - [Angular wraps](https://docs.angularjs.org/api/ng/service/$window) native objects for namespacing, which makes testing easier.

```suggestion
            $window.scrollTo(0,0);
```